src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedOrganizationConnection.java

```diff
@@ -47,20 +49,26 @@ public class VersionedOrganizationConnection extends VersionedConnection {
    */
   private static final String ORGANIZATIONS = "/organizations";
   private static final String ORGANIZATIONS_BRANDS = "/organizationBrands";
+  private static final String ORGANIZATION_ACLS = "/organizationAcls";
   
   /**
    * Keys
    */
   private static final String VANITY_NAME_KEY = "vanityName";
   private static final String EMAIL_DOMAIN_KEY = "emailDomain";
   private static final String PARENT_KEY = "parent";
+  private static final String ROLE_KEY = "role";
+  private static final String STATE_KEY = "state";
+  private static final String ORGANIZATION_KEY = "organization";
   
   /**
    * Param value
    */
   private static final String VANITY_NAME_VALUE = "vanityName";
   private static final String EMAIL_DOMAIN_VALUE = "emailDomain";
   private static final String PARENT_ORGANIZATION_VALUE = "parentOrganization";
+  private static final String ROLE_ASSIGNEE_VALUE = "roleAssignee";
+  private static final String ORGANIZATION_VALUE = "organization";
```

Comment:
just checking that it is correct that the organization key and value both being `"organization"` is correct?

Reply:
yes. Referring to https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/organizations/organization-access-control-by-role?view=li-lms-2022-11&tabs=http#find-organization-administrators
`GET https://api.linkedin.com/rest/organizationAcls?q=organization&organization={encoded organization URN}&role=ADMINISTRATOR&state=APPROVED`